### ISSUE TITLE
Dependency versions not specified

### ISSUE THREAD

**Issue by sg-** [mbartling/homebrew-mbed-cli#1]
Doesn't seem like this specifies versions of dependencies, should it?

**Comment by mbartling** [comment]
No, it currently fetches the latest on ci check. Adding the versioning is easy, I just didn't want to maintain it

**Comment by mbartling** [comment]
@sg- I just added support for versioning, starting with 1.5.1

```
&> brew install mbed-cli@1.5.1
```

The versions are updated automatically any time CI is triggered
